tensorflow_addons/layers/wrappers.py

```diff
@@ -118,34 +78,77 @@ def build(self, input_shape):
             self.kernel_norm_axes = list(
                 range(self.layer.kernel.shape.rank - 1))
 
-            self.layer.v = self.layer.kernel
-            self.layer.g = self.layer.add_variable(
+            self.v = self.layer.kernel
+            self.g = self.add_variable(
                 name="g",
                 shape=(self.layer_depth,),
                 initializer=tf.keras.initializers.get('ones'),
                 dtype=self.layer.kernel.dtype,
-                trainable=True,
-                aggregation=tf.VariableAggregation.MEAN)
```

Comment:
Should we be dropping the variable aggregation? We have no way to test this is distributed fashion, but [based on this PR](https://github.com/seanpmorgan/tensorflow/pull/1) I believe its useful.

Reply:
For trainable weights, usually the aggregation is handled by the distribution strategy scope, which is why we always leave the aggregation to be default value. If you check the code base, the aggregation is only specified for non-trainable weights, eg in metric and batch norm. Those non trainable weights are not updated in backprop, and handled specially in forward pass. 

Having aggregation for trainable variable is likely to cause issue with the combination of distribution strategy, and the code owner is better to be sure what they are trying to do.

Reply:
Thank you that makes sense. So regarding the error the user mentioned in that PR... should they have set a global aggregation strategy or something? I would have assumed it'd default to something instead of erroring like they mentioned?

Reply:
Its unclear to me what the error/warning message is from that PR. The default aggregation is None when no distribution strategy is specified. For individual distribution strategy, the reduction is different, and the default value should be available in the API doc.